lib/bundler/shared_helpers.rb

```diff
@@ -19,6 +19,16 @@ def requirement
 
 module Bundler
   module SharedHelpers
+    def default_root
+      given_gemfile = ENV["BUNDLE_GEMFILE"]
+      return Pathname.new(given_gemfile).dirname if given_gemfile && !given_gemfile.empty?
+      search_up('') do |path|
+        if path.join("Gemfile").file? || path.join("gems.rb").file? || path.join(".bundle").directory?
```

Comment:
Okay, so I should put the check from `SharedExamples#default_bundle_dir` here, right?